src/scripts/clientUtils.js

```diff
@@ -49,16 +55,9 @@ export function onServerChanged(userId, accessToken, apiClient) {
 
 export function logout() {
     window.connectionManager.logout().then(function () {
-        let loginPage;
-
-        if (AppInfo.isNativeApp) {
-            loginPage = 'selectserver.html';
-            window.ApiClient = null;
```

Comment:
Shouldn't `ApiClient` be nulled?

Reply:
I don't believe so, it seems to work without this.